content/en/docs/contribute/intermediate.md

```diff
@@ -379,25 +384,30 @@ After you decide which branch to start your work (or _base it on_, in Git
 terminology), use the following workflow to be sure your work is based on the
 most up-to-date version of that branch.
 
-1.  Fetch both the `upstream` and `origin` remotes. This updates your local
-    notion of what those branches contain, but does not change your local
-    branches at all.
+1.  There are three different copies of the repository when you work locally:
+    `local`, `upstream`, and `origin`. Fetch both the `origin` and `upstream` remotes. This
+    updates your cache of the remotes without actually changing any of the copies.
 
       ```bash
-      git fetch upstream
       git fetch origin
+      git fetch upstream
       ```
 
-2.  Create a new tracking branch based on the branch you decided is the most
-    appropriate. This example assumes you are using `master`.
+    This workflow deviates from the one defined in the Community's [GitHub
+    Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md).
+    You are not merging your local copy of `master` with `upstream/master` and then
+    pushing the updates to your fork. That step is not required in
+    `kubernetes/website` because you are not building from your fork.
+
+2.  Create a local working branch based on the branch you decided is the most
+    appropriate. This example assumes you are using `master`. Because you didn't
```

Comment:
```suggestion
    This example assumes you are basing work on `upstream/master`. Because you didn't
```

Reply:
@zacharysarah - since you are suggesting changes to the original text that I didn't modify (merely renumbered), going forward should I reword original text for clarity?